pkg/motor/query.go

```diff
@@ -82,34 +38,33 @@ func (mtr *motorNodeImpl) handleBucketGroupQuery(addr string) (*mt.QueryResultIt
 		return nil, err
 	}
 
-	return &mt.QueryResultItem{
-		Kind:        common.EntityKind_ADDRESS,
-		WhereIsList: ptrArr,
+	return &mt.QueryWhereIsByCreatorResponse{
+		Code:    http.StatusAccepted,
+		WhereIs: ptrArr,
 	}, nil
 }
 
-func (mtr *motorNodeImpl) handleRegistryQuery(did string) (*mt.QueryResultItem, error) {
-	resp, err := mtr.GetClient().QueryWhoIs(did)
+func (mtr *motorNodeImpl) QueryRegistry(req mt.QueryWhoIsRequest) (*mt.QueryWhoIsResponse, error) {
```

Comment:
nit: name this `QueryWhoIs` as it's more accurate, and more things could potentially be added to registry in the future creating ambiguity.

Reply:
i would agree, but at this time its the only thing we are querying from the registry module. to keep it consistent i went with that name.